src/test_utils.jl

```diff
@@ -0,0 +1,56 @@
+"""
+    FeatureTransforms.TestUtils
+
+Provides fake [`Transform`](@ref)s for testing purposes only.
+"""
+
+module TestUtils
+
+using ..FeatureTransforms
+using ..FeatureTransforms: OneToOne, OneToMany, ManyToOne, ManyToMany
+using Tables
+
+export FakeOneToOneTransform, FakeOneToManyTransform
+export FakeManyToOneTransform, FakeManyToManyTransform
+
+for C in (:OneToOne, :OneToMany, :ManyToOne, :ManyToMany)
+    FT = Symbol(:Fake, C, :Transform)
+    @eval begin
+        """
+            $($FT) <: Transform
+
+        A fake [`$($C)`](@ref) transform for test purposes.
+        Calling `apply` will return an array of ones with the expected size and dimension.
```

Comment:
```suggestion
        Calling `apply` will return an array of ones with a size and dimension matching the cardinality of the transform.
```
(note line length)

"expected size" sounds a bit strange because the exact size doesn't matter